### ISSUE TITLE
Launch GraphQL Foundation Monthly Newsletter

### ISSUE THREAD

**Issue by caniszczyk** [graphql/foundation#26]

**Comment by Urigo** [comment]
@caniszczyk What should be the next steps?
Maybe it's worth reaching our to Prisma about their [GraphQL Weekly newsletter](https://www.graphqlweekly.com)?

They've done great work there but it seems to be a bit slow recently and maybe collaborating on that could be a good idea?

**Comment by IvanGoncharov** [comment]
> Maybe it's worth reaching our to Prisma about their GraphQL Weekly newsletter?

@Urigo I think it's a great idea. 
AFAIK it's currently maintained by Novvum guys and they are members of the foundation.

**Comment by Urigo** [comment]
@IvanGoncharov do you know their Github usernames?  Can you tag them here?

**Comment by Urigo** [comment]
cc @rohit-ravikoti

**Comment by rajinwonderland** [comment]
Hey @Urigo! Yes! We have been a bit understaffed lately, so haven't had much time to update it, but shouldn't take much to flip it over to the foundation. How should we proceed with this?

**Comment by Urigo** [comment]
@rajinwonderland that's amazing and extremely generous!!

I think @brianwarner & @caniszczyk should follow with next steps?

**Comment by rajinwonderland** [comment]
Sweet! We'll be had to help in anyway possible 👍

**Comment by brianwarner** [comment]
Yep! Raj, I'll connect with you separately.

**Comment by caniszczyk** [comment]
We have started a monthly newsletter draft here: https://docs.google.com/document/d/14ZwpLnIuiSnHfD0zSb2gLoC30MirG1LHlpcF3LFsPFM/edit?ts=5f4fbf18

The idea is to do this in the open completely and get feedback from folks before posting.

**Comment by benjie** [comment]
I was working on the Tagged type RFC just earlier today as it happens. Can I suggest text a little more like this? (Feel free to add links/styles/brevity/otherwise edit as appropriate.)


### Tagged Type RFC

This is an RFC for a new "Tagged" type to be added to GraphQL, to help address the input polymorphism feature gap. Originally proposed as the `@oneField` directive, the Tagged type has evolved into an entire new type in the type system, and is currently the lead proposal from the Input Unions Working Group.
 
A Tagged type defines a list of named member fields each with an associated type (like the fields in Object and Input Object types); but with the stipulation that exactly one of these fields must be present (no more, no less). Depending on the types of the member fields, a Tagged type could be appropriate for input, output, or both, which is proving to be quite controversial; it's possible that the tagged type might be scoped to input only before release. The Tagged type was most recently introduced to the GraphQL Working Group in August and is undergoing feedback and revision before an expected update with the working group in October or November.

If you have use-cases for input polymorphism, please take a look - feedback is very welcome.

**Comment by benjie** [comment]
Separately, can I suggest re-ordering of the items.

1. GraphQL over WebSockets refresh - top priority because this affects all the current users of GraphQL over websockets, and it would be great to get buy-in on the new language-agnostic spec from the community (and also it needs more eyes for security, performance and scalability reviews)
2. Tagged type RFC - next highest priority because it's a significant change to the GraphQL spec and needs more eyes
3. Field coordinates - a relatively minor proposal about standardizing how we refer to things in GraphQL; it's useful for all GraphQL people to know about this, but not as important as the two above
4. DocumentNode generics - I, personally, am very excited for this! But it's very TypeScript specific, so I think all of the above have higher priority to the GraphQL community as a whole; sorry @dotansimha :grimacing:

**Comment by IvanGoncharov** [comment]
@caniszczyk A few suggestions from me:
* @hereisnaman Successfully finished he's GSoC project and we in process of merging his PR https://github.com/graphql/graphql-js/pull/2770. Here some context: https://foundation.graphql.org/news/2020/06/30/google-summer-of-code-2020-participant-naman/
* We were accepted to for GSoD and @carolstran will be working on the FAQ section of graphql.org https://developers.google.com/season-of-docs/docs/participants/project-graphql-carolstran
* "GraphQL over HTTP" was moved under graphql foundation https://github.com/graphql/graphql-over-http 
* @lilianammmatos and @robrichard having big progress on `@deferer & @stream`. Here is their talk about it from recent GraphQLSummit: https://www.youtube.com/watch?v=icv_Pq06aOY
* @andimarek and @eapache started https://github.com/graphql/graphql-scalars/issues initiative.
Andy's also gave a talk about it during GraphQL Summit: https://www.youtube.com/watch?v=SRGTaYL3h9c
* All WG recording were published on Youtube https://www.youtube.com/channel/UCERcwLeheOXp_u61jEXxHMA
And we already have 20 subscribers 🎉

**Comment by Urigo** [comment]
Thank you @caniszczyk for the update!

1. When are you planning to publish the newsletter?
2. Also what would be the cadence, monthly?
3. I do think the newsletter is a great platform to highlight the work that has been done on the graphql-js library (and all the other Foundation assets). It also is probably the only platform.  So maybe we could have section like "Last month on the foundation's projects" and have a few bullets points on what happened.  There we could also highlight TypedDocumentNode among others
4. @brianwarner could you update on the conversation you had with @rajinwonderland and what was the result?

**Comment by Urigo** [comment]
Also adding @acao to share more work from GraphiQL and stuff

**Comment by Urigo** [comment]
also - can we add a place to accumulate things between newsletters? 
like a backlog?

**Comment by rajinwonderland** [comment]
Hey@Urigo, I think @brianwarner most likely had a conversation with Rohit @rohit-ravikoti.

If there's anything I can answer though, I'd be happy to.

**Comment by mayaptos** [comment]
Hi All - I will be putting the blog together each month and your feedback and contributions are extremely helpful. Keep it up!

@benjie Thanks for the re-order and the updated text

@IvanGoncharov Great stuff. Thanks for contributing

@Urigo - We plan to publish ASAP and the cadence will be monthly. Given the Sept WG meeting just ocured the next one will post sooner, but in general it'll be a monthly cadence.

Happy to add more info on projects. That would be great. They just need to be supplied to me and we will add them.

**Comment by mayaptos** [comment]
All of your feedback and contributions has been added to the blog. Let me know if we are missing anything. otherwise we will get this posted early next week! 
https://docs.google.com/document/d/14ZwpLnIuiSnHfD0zSb2gLoC30MirG1LHlpcF3LFsPFM/edit?ts=5f46a3be#

**Comment by Urigo** [comment]
thank you @mayaptos !
I've added a couple of suggestion to the doc itself.

**Comment by Urigo** [comment]
and I still hope that @acao will get to add his updates as well, but I can't add them myself

**Comment by Urigo** [comment]
another question - on which channels will you spread the newsletter?
and how can people subscribe?
Will you use the official social accounts to promote it?

**Comment by caniszczyk** [comment]
1) we will post to the blog here https://foundation.graphql.org/news/
2) we will setup a newsletter form using an internal LF system (we have marketing cloud behind the scenes)
3) yes, we'd love too!

The goal is to start doing this every month and solicit input from the community

**Comment by mayaptos** [comment]
Thanks @Urigo ! Could we get a link back to the newsletter. I don't see one on the Graph weekly site

**Comment by Urigo** [comment]
@mayaptos yes I saw that now, in looks different on the newsletter itself and I've mentioned it to @chenkie.

He said they are working on it, @chenkie could you give on update about that when you'll get a chance?

In the meantime, I'll try to put here some screenshots from the actual newsletter here for review:

<img width="1075" alt="Screen Shot 2020-10-31 at 11 38 24 AM" src="https://user-images.githubusercontent.com/1297448/97776013-b4954a00-1b6d-11eb-8dd1-daf4a4b62bde.png">
<img width="1147" alt="Screen Shot 2020-10-31 at 11 38 33 AM" src="https://user-images.githubusercontent.com/1297448/97776012-b2cb8680-1b6d-11eb-8162-53e64853093e.png">
<img width="1079" alt="Screen Shot 2020-10-31 at 11 38 45 AM" src="https://user-images.githubusercontent.com/1297448/97776008-ae9f6900-1b6d-11eb-8624-b5bf817da23a.png">

**Comment by Urigo** [comment]
another issue was that they could current only have one link per items and not links in the texts.
I hope they could improve on that as well soon because it helped create better context.

**Comment by chenkie** [comment]
Outstanding items that I'm working on to round out the GraphQL Weekly Foundation Edition newsletter:

* Description from top of email in website as well (which includes link back to Foundation)
* Support for links in description body
